packages/rest/src/lib/REST.ts

```diff
@@ -0,0 +1,134 @@
+import { deepClone, mergeDefault } from '@sapphire/utilities';
+import { EventEmitter } from 'events';
+import { CDN } from './CDN';
+import { InternalRequest, RequestData, RequestManager, RequestMethod } from './RequestManager';
+import { DefaultRestOptions } from './utils/constants';
+
+/**
+ * The events that the REST manager emits
+ */
+export const enum RESTEvents {
+	Debug = 'restDebug',
+	RateLimited = 'rateLimited',
+}
+
+/**
+ * Options to be passed when creating the REST instance
+ */
+export interface RESTOptions {
+	/**
+	 * The base api path, without version
+	 *
+	 * @default 'https://discord.com/api'
+	 */
```

Comment:
i don't think there should be a newline here, this is rather inconsistent across the pr